From 466367eb5ebc4ea1b9b1457d496834c57ed1dd66 Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Thu, 26 Mar 2020 23:41:59 +0800
Subject: [PATCH] syslog-ng.conf: fix the source

ERROR: Using /dev/log Unix socket with systemd is not possible.

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
---
 syslog-ng-config/files/syslog-ng.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/syslog-ng-config/files/syslog-ng.conf b/syslog-ng-config/files/syslog-ng.conf
index b1bab2c..1e06c02 100644
--- a/syslog-ng-config/files/syslog-ng.conf
+++ b/syslog-ng-config/files/syslog-ng.conf
@@ -62,7 +62,7 @@ options { chain_hostnames(off); flush_lines(0); use_dns(no); use_fqdn(no);
 # This is the default behavior of sysklogd package
 # Logs may come from unix stream, but not from another machine.
 #
-source s_src { unix-dgram("/dev/log"   ); internal();
+source s_src { systemd_journal(); internal();
                file("/proc/kmsg" program_override("kernel")   );
 };
 
-- 
2.7.4

